client/src/App.js

```diff
@@ -16,42 +16,43 @@ import ResetPassword from './pages/ResetPassword';
 
 const App = () => {
   return (
-    <Provider store={store}>
-      <PersistGate loading={null} persistor={persistor}>
-        <Router>
-          <Header />
-          <div className="container">
-            <Switch>
-              <Route path="/" exact component={() => <Home />} />
-              <Route
-                path="/activate/:id"
-                component={() => <ActivateAccount />}
-              />
+    <div className="min-vh-100 d-flex flex-column">
```

Comment:
Let providers to be outer most component and this div as a child for the provider.
Reason, code clarity.

Reply:
Fixed